languages/csharp/README.md

```diff
@@ -1,260 +1,17 @@
-_Note: All subconcepts should be split out into separate files under the concepts directory and hyperlinked._
-
 # C&#35;
 
 C# is an object-oriented language.
```

Comment:
I'd probably remove this and instead start with "Thanks for wanting to contribute to the C# track" etc. 

"C# is an object-oriented language." probably fits better in the `reference.md` file

Reply:
It does. This was just placeholder content :)